### ISSUE TITLE
Make an under construction template

### ISSUE THREAD

**Issue by kjaymiller** [code-newbies/python-thursday-blog#22]
<b>Derek Gustafson</b> added the card <a href="http://ift.tt/1JFzB0U">Make an under construction template</a> to the <b>Issues</b> list in the <b>PTBlog</b> board at September 3, 2015 at 06:18PM<br><br><br><br><a href="http://ift.tt/1JFzB0U">View on Trello</a>

**Comment by andrealmar** [comment]
I think we can close this issue cause we are already working on bootstrap layout

**Comment by tgmiller5** [comment]
I noticed the under construction template when running it

**Comment by andrealmar** [comment]
Yep but with the bootstrap layout there is no need for an under construction template anymore
